documentation/modules/proc-iot-configure-hono-cli.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// assembly-iot-getting-started.adoc
+// assembly-iot-tenant-guide.adoc
+
+[id='iot-cli-{context}']
+
+ifeval::["{cmdcli}" == "oc"]
+:http-adapter: $(oc -n enmasse-infra get routes iot-http-adapter --template='{{ .spec.host }}')
+:mqtt-adapter: $(oc -n enmasse-infra get routes iot-mqtt-adapter --template='{{ .spec.host }}')
+:mqtt-port: 443
+endif::[]
+ifeval::["{cmdcli}" == "kubectl"]
+:http-adapter: $(kubectl -n enmasse-infra get service iot-http-adapter-external -o jsonpath={.status.loadBalancer.ingress[0].hostname}):30443
+:mqtt-adapter: $(kubectl -n enmasse-infra get service iot-mqtt-adapter-external -o jsonpath={.status.loadBalancer.ingress[0].hostname})
+:mqtt-port: 30883
+endif::[]
+
+== Installing command-line client
+
+.Prerequisites
+* link:{BookUrlBase}{BaseProductVersion}{BookNameUrl}#installing-services-messaging-iot[The IoT services are installed].
+* link:{BookUrlBase}{BaseProductVersion}{BookNameUrl}#iot-creating-project-messaging-iot[An IoT project is created].
+* link:{BookUrlBase}{BaseProductVersion}{BookNameUrl}#iot-creating-device-messaging-iot[An IoT device is created].
+
+.Procedure
+
+. Download the link:https://www.eclipse.org/hono/downloads/[Eclipse Hono command-line client].
```

Comment:
This is something which we might want to discuss. As we wouldn't have the Hono client downstream.

Reply:
Yeah. This is just a refactoring of the current material. That's a job for another PR. We definitely want to see how to improve developer experience.